src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java

```diff
@@ -80,7 +80,33 @@ public Boolean apply(WebDriver from) {
       @Override
       public String toString() {
         return String.format("value ('%s') to be present in element found by %s", value,
-            element.getTagName());
+                             element.getTagName());
+      }
+    };
+  }
+
+  /**
+   * An expectation for checking if the given text is present in the specified element.
+   *
+   * @author Michal Nowierski
+   */
+  public static ExpectedCondition<Boolean> valueToBeNotPresentInElementsAttribute(
+      final WebElement element, final String attribute, final String value) {
+
+    return new ExpectedCondition<Boolean>() {
+      public Boolean apply(WebDriver from) {
+        try {
+          String elementsAttributeValue = element.getAttribute(attribute);
+          return !elementsAttributeValue.contains(value);
+        } catch (StaleElementReferenceException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)